modules/claims_api/spec/requests/metadata_spec.rb

```diff
@@ -85,6 +86,22 @@
             end
           end
         end
+
+        local_bgs_itf_services = %i[intenttofile]
+        local_bgs_itf_methods = %i[insert_intent_to_file]
+        local_bgs_itf_services.each do |local_bgs_itf_service|
+          it "returns the correct status when the local bgs #{local_bgs_itf_service} is not healthy" do
+            local_bgs_itf_methods.each do |local_bgs_itf_method|
+              allow_any_instance_of(ClaimsApi::IntentToFileWebService).to receive(
+                local_bgs_itf_method.to_sym
+              )
+                .and_return(Struct.new(:healthy?).new(false))
+              get "/services/claims/#{version}/upstream_healthcheck"
+              result = JSON.parse(response.body)
+              expect(result["localbgs-#{local_bgs_itf_service}"]['success']).to eq(false)
+            end
+          end
+        end
```

Comment:
@mchristiansonVA  I had a similar comment on another PR for this kind of update so just asking here too.  Is this a loop because we expect `local_bgs_itf_services` to grow in the future?  I assume the methods can and will grow but will the services?

Perhaps more of a nit really, but just want to understand the changes.

Reply:
I based that on [this test](https://github.com/department-of-veterans-affairs/vets-api/blob/3c9306f2484d93544408732b63ae7746cf4d1c76/modules/claims_api/spec/requests/metadata_spec.rb#L75), which had a similar approach, but you're right we don't need the loop on the services part here.